MountSionCBSSecondary/SchoolWebsite
# Add semantic themes to website

# Description

Added a local version of semantic ui to allow the use of the overriding of variables to add customization to the semantic CSS.

## Verification Steps

*IMPORTANT* To verify this PR it should work as normal load the index.html in your favorite browser to see the mount sion blue nav bar. 

### EDIT: New Steps to Verify or Change Semantic CSS
If you wish to customize semantic and play around with the themes you will need to follow the following steps.

- Install [Node](https://nodejs.org/en/) (Latest long term support version)
Once you got node installed you will need to install `gulp` via the node package manager:
```
$ npm install -g gulp
```
With gulp installed globally navigate to the root directory of the project and run the following command in order to install the required node modules for semantic
```
$ npm run build:semantic
```
Now you are ready to go. If you navigate to `semantic/src/site/site.variables` here you can edit the color of blue or set other variables, see the semantic [docs](https://semantic-ui.com/usage/theming.html) for more info on this.
Any changes you make to these variables will require you to rebuild the semantic CSS and this can be done in the following directory `semantic/` by running:
```
$ gulp build
```
This will kick off the build and then you will see the changes in your browser, just use semantic as normal in the regular html.

I will document the steps in another PR, it is import to get this merged as soon as possible to allow all order PR's get rebased off of and get use of the new colors and any other changes we want to make to semantic

![theme-nav](https://user-images.githubusercontent.com/14313111/39986087-84aeb726-5758-11e8-8ada-906c1f059fd2.png)
